probe-rs/src/domain.rs

```diff
@@ -0,0 +1,20 @@
+pub struct CountingIter<'a, I>(I, &'a mut usize);
```

Comment:
As it turns out, this module is not only unused, it's not even declared :)

Reply:
Yeah I think it fell through the cracks :)